### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by pankaj-prog**
add: 
-  carousel
-  categories
-  style: sidebar 
-  videos database and images

preview: 
BlitheBrains - homepage

Files to check: 
1. home.jsx
2. categoreisSection.jsx
3. categorycard
4. carousel
5. carouselCard

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
- Loved the UI. It's clean.
- Code is very neat and easy to read.
- Everything looks really nice. Good to merge.

File: src/components/Carousel/CarouselCard.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const CarouselCard = ({ video }) => {
+  return (
+    <article className="carousel-card">
+      <img src={video.thumbnail} alt={video.title} />
+      <button
+        className="btn btn-solid-secondary btn-rc"
+        onClick={() => console.log("clicked")}
+      >
```
Comment (bhtibrewal) [inline src/components/Carousel/CarouselCard.jsx]: Remove console.log

**Review by bhtibrewal — APPROVED** [review]
This is a small pr couldn't find any mistakes, you are good to merge

File: src/components/Carousel/CarouselCard.jsx (hunk 1 above)
Comment (pankaj-prog) [reply src/components/Carousel/CarouselCard.jsx]: Have't not implemented the functionality part yet, will remove it and replace it with other method

**Pull request merged**